HandsOn/Group17/mappings/yarrrml_mapping.yml

```diff
@@ -0,0 +1,115 @@
+prefixes:
+  rr: http://www.w3.org/ns/r2rml#
+  foaf: http://xmlns.com/foaf/0.1/
+  xsd: http://www.w3.org/2001/XMLSchema#
+  rdfs: http://www.w3.org/2000/01/rdf-schema#
+  rev: http://purl.org/stuff/rev#
+  schema: http://schema.org/
+  dct: http://purl.org/dc/terms/
+  rml: http://semweb.mmlab.be/ns/rml#
+  ql: http://semweb.mmlab.be/ns/ql#
+  rdf: http://www.w3.org/1999/02/22-rdf-syntax-ns#
+  vocab: http://example.org#
+  
+mappings:
+
+  triplesmappuntodemuestreo:
+    sources:
+      - [abr_mo19-updated.csv~csv]
+    s: http://www.wsld.org/grupo17/ontology/$(PUNTO_MUESTREO)
+    po:
+      - [a, vocab:esPunto]
+      - [vocab:esPunto,$(PUNTO_MUESTREO)]
+      - [vocab:esPunto2,$(http://www.wsld.org/grupo17/ontology/esPuntoMuestreo)~iri]
+      - [vocab:esPunto3,$(string-column),xsd:string]
+      - p: vocab:mide
```

Comment:
1) syntax is not correct
2) you need to specify the conditions.

Let me put an example

```
- p: vocab:estaEnEstacion
  o: 
    - mapping: triplesmapestacion
      condition:
          function: equal
          parameters:
             - [str1, $(ESTACION)]
             - [str2, $(ESTACION)]
```
be careful with the indentations in YAML! 

y así para el resto de joins
el resto está correcto